data/crd/ssp.kubevirt.io_ssps.yaml

```diff
@@ -0,0 +1,862 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Do we want to commit this file, or is it enough that it is generated by `make container-build`?
@omeryahud, You have mentioned that these files are needed for downstream, but do they need to be commited in the repo?

Reply:
It should be committed because we don't have `operator-sdk` downstream to generate manifests, we can think of a way to resolve it in the future, but to quickly resolve this for now, they should be committed

Reply:
Ok.